### DISCUSSION TITLE
Page scroll interrupted by one-line code block

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19909] [Projects and Issues] [answered]
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/106575310/177021005-38c66187-73f0-43b6-9be0-b093e4bcff20.png">

url: https://github.com/pmndrs/zustand#fetching-everything

The scrollbar appears, when I move my cursor to the one-line code block. I cannot scroll up and down the whole page. 
The scroll event does not propagate?

Multiple-line code blocks do not have scrollbars. Maybe we should make this one-line code block higher to avoid the scrollbar?

**Comment by diegoleondev** [comment, marked as answer]
Hello, possibly it is a bug or browser style, in my case the scroll does not appear.

![image](https://user-images.githubusercontent.com/76979144/177058016-5ab114e3-5589-4545-9b99-f7ee514ef44e.png)

**Reply by ghost** [reply]
@dalldev Sorry my bad. You're right. I found the problem. I am replacing the code font with a chrome extension called `Stylish`.
